task.sql

```diff
@@ -4,30 +4,66 @@
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 );
 
+CREATE TABLE Cities (
+		ID INT AUTO_INCREMENT,
+		Name VARCHAR(50),
+		CountryID INT,
+		FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
+		PRIMARY KEY (ID)
+);
+
+CREATE TABLE Addresses (
+		ID INT AUTO_INCREMENT,
+		StreetName VARCHAR(100),
+		CityID INT,
+		FOREIGN KEY (CityID) REFERENCES Cities(ID) ON DELETE NO ACTION,
```

Comment:
Similar to the previous comment, using `ON DELETE NO ACTION` for the foreign key constraint might cause issues if a city is deleted while addresses still reference it. Consider using `ON DELETE CASCADE` if appropriate.